incubator/java-spring-boot2/stack.yaml

```diff
@@ -1,5 +1,5 @@
 name: Spring Boot®
-version: 0.3.30
```

Comment:
I know this specific change was discussed, but now looking at it, wouldn't these stack.yaml versions be bumped to x.x.x+1?